### PR TITLE
Appease clippy

### PR DESCRIPTION
**Pull request by lkatalin**
Signed-off-by: Lily Sturmann <lsturman@redhat.com>

### CODE REVIEW

File: src/ima_entry.rs [hunk 1]
```diff
@@ -20,7 +20,7 @@ pub trait EncodeLegacy {
     fn encode_legacy(&self, writer: &mut dyn Write) -> Result<()>;
 }
 
-#[derive(PartialEq)]
+#[derive(PartialEq, Eq)]
```
Comment (ueno) [inline src/ima_entry.rs:23]: Is `PartialEq` still needed, given `Eq` is defined as: `trait Eq: PartialEq`?

**Review by ueno — APPROVED** [review]
Looks good to me.

File: src/ima_entry.rs (hunk 1 above)
Comment (lukehinds) [reply src/ima_entry.rs:23]: I have seen clippy complain about this a few times (even when it breaks code), if you prefer to shut it down this is the ignore option:

https://github.com/sigstore/sigstore-rs/pull/102/files

Comment (lkatalin) [reply src/ima_entry.rs:23]: @ueno I could update this to just derive `Eq`

Comment (lkatalin) [reply src/ima_entry.rs:23]: Looks like the compiler does not like it:
```
[lily@localhost rust-keylime]$ cargo build
   Compiling keylime_agent v0.1.0 (/home/lily/Repos/rust-keylime)
error[E0277]: can't compare `ima_entry::Digest` with `ima_entry::Digest`
  --> src/ima_entry.rs:23:10
   |
23 | #[derive(Eq)]
   |          ^^ no implementation for `ima_entry::Digest == ima_entry::Digest`
   |
   = help: the trait `PartialEq` is not implemented for `ima_entry::Digest`
note: required by a bound in `Eq`
   = note: this error originates in the derive macro `Eq` (in Nightly builds, run with -Z macro-backtrace for more info)
help: consider annotating `ima_entry::Digest` with `#[derive(PartialEq)]`
   |
24 | #[derive(PartialEq)]
   |

```

Comment (ueno) [reply src/ima_entry.rs:23]: Thanks for trying; let's keep it as it is.

**Pull request merged**